README.md

```diff
@@ -1,10 +1,11 @@
 
-# Guess the number
+# Guess The Number
 
-**Simple logic game**
-  
-**Program is a simple number guessing game, which looked something like this:**
+**This is a simple "Guess The Number" game where the user has to guess a number randomly generated by the computer.**
 
+This repository has implementations in C++, JavaScript and Java.
```

Comment:
You can also try to add `badge` if you have time. As it makes project looks cool and friendly  

You can take these badges from here https://forthebadge.com/ 
These are free to use without giving credits. You can learn how to add images in in markdown with image links.